en/prepare/gnss.rst

```diff
@@ -0,0 +1,122 @@
+******************************
+GNSS and precision measurement
+******************************
+
+QField can make use of the internal GNSS (Global Navigation Satellite System, like GPS, GLONASS, Galileo or Beidou).
+QField can also connect to external antennas through bluetooth and directly integrate their NMEA stream.
+
+Quality indication
+==================
+
+Additional variables
+....................
+
+You can get access to positioning information through additional expression variables.
+These will only be available when positioning is enabled.
+
+These variables are commonly used as default values for fields to keep track of the quality of
+individual measured points.
+
+- `@position_coordinate`
+  - A point with the coordinate in WGS84. Lon, Lat, Altitude as delivered by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - `x(@position_coordinate)`
+  - IE
+- `@position_timestamp`
+  - The timestamp of the position in UTC as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_direction`
+  - The direction of movement in degrees from true north as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_ground_speed`
+  - Groundspeed (in m/s) as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_magnetic_variation`
+  - The angle between the horizontal component of the magnetic field and true north, in degrees as reported by the sensor. Also known as magnetic declination. A positive value indicates a clockwise direction from true north and a negative value indicates a counter-clockwise direction. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_horizontal_accuracy`
+  - The horizontal accuracy of the coordinate (in meters) as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_vertical_accuracy`
+  - The vertical accuracy of the coordinate (in meters) as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_3d_accuracy`
+  - The 3 dimensional accuracy of the coordinate (in meters), 3D-RMS as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE
+- `@position_vertical_speed`
+  - The vertical speed (in m/s) as reported by the sensor. It is only available when the crosshair is snapped to the sensor.
+  - IE 
+- `@position_source_name`
```

Comment:
I would move that to the very top, because it explains how all the other @position_* variables behave really.